test/net/sourceforge/kolmafia/request/FightRequestTest.java

```diff
@@ -1077,6 +1078,26 @@ public void canDetectLilDoctorBagUses() {
     }
   }
 
+  @Test
+  public void canDetectEvilometerChange() {
+    AdventureResult item = ItemPool.get(ItemPool.LIHC_EYE, 1);
+    assertEquals(0, item.getCount(KoLConstants.inventory));
+    var cleanups = new Cleanups(withFight(), withProperty("cyrptNicheEvilness", 50));
+    try (cleanups) {
+      assertThat("cyrptNicheEvilness", isSetTo(50));
+      String html = html("request/evilometer_fight.txt");
```

Comment:
Can you rename this file `test_fight_evilometer_single_loud_beep.html` or something